### DISCUSSION TITLE
Supporting React in Justpy

### DISCUSSION THREAD

**Discussion by bigfatwhale** [elimintz/justpy#273] [General]
Just out of curiosity, is it technically possible to have something like justpy for react?

**Comment by elimintz** [comment]
Maybe, you need a front end that does not need to be built using npm. That is why I choose Vue originally. Not sure but maybe it can be done with react.

**Comment by WolfgangFahl** [comment]
I think the question is too general. Could you please provide a specific react use case that you'd like to see implemented?
